scripts/schema-test-coverage.mjs

```diff
@@ -110,6 +119,68 @@ const runTests = async (schemaUri, testDirectory) => {
   };
 };
 
+addKeyword({
+  id: "https://spec.openapis.org/oas/schema/vocab/keyword/discriminator",
+  interpret: (discriminator, instance, context) => {
+    return true;
+  },
+  /* discriminator is not exactly an annotation, but it's not allowed
+   * to change the validation outcome (hence returing true from interopret())
+   * and for our purposes of testing, this is sufficient.
+   */
+  annotation: (discriminator) => {
+    return discriminator;
+  },
+});
+
+addKeyword({
+  id: "https://spec.openapis.org/oas/schema/vocab/keyword/example",
+  interpret: (example, instance, context) => {
+    return true;
+  },
+  annotation: (example) => {
+    return example;
+  },
+});
+
+addKeyword({
+  id: "https://spec.openapis.org/oas/schema/vocab/keyword/externalDocs",
+  interpret: (externalDocs, instance, context) => {
+    return true;
+  },
+  annotation: (externalDocs) => {
+    return externalDocs;
+  },
+});
+
+addKeyword({
+  id: "https://spec.openapis.org/oas/schema/vocab/keyword/xml",
+  interpret: (xml, instance, context) => {
+    return true;
+  },
+  annotation: (xml) => {
+    return xml;
+  },
+});
+
+defineVocabulary(
+  "https://spec.openapis.org/oas/3.1/vocab/base",
```

Comment:
This needs to be parameterized with the OpenAPI version of the branch it is running on.

No spontanous idea on how to reliably do that, other than

Reply:
Which would end up in main (and then dev and v3.1-dev) after the release unless someone fiddles with the v3.2.0-rel branch.

Automatically detecting the release seems preferable.

Reply:
@ralfhandl Which is fine but let's fix it separately.  I do not have the mental bandwidth today for that, and want to get the XML PR merged so I can unblock further work that depends on it.

Reply:
> This apparently needs to be parameterized with the OpenAPI version of the branch it is running on

I didn't realize you were versioning the vocabulary now. So, ignore what I said about not needing `defineVocabulary`, but you still shouldn't need to implement the keywords as long as you use the correct import.